src/maps/applicable-facility-attributes.map.ts

```diff
@@ -11,17 +11,14 @@ export class ApplicableFacilityAttributesMap extends BaseMap<
 > {
   public async one(entity: any): Promise<any> {
     return {
-      [propertyMetadata.year.fieldLabels.value]: Number(entity.pyd_op_year),
-      [propertyMetadata.programCode.fieldLabels.value]: entity.pyd_prg_code,
-      [propertyMetadata.facilityId.fieldLabels.value]: Number(
-        entity.uf_orispl_code,
-      ),
-      [propertyMetadata.stateCode.fieldLabels.value]: entity.uf_state,
-      [propertyMetadata.unitTypeCode.fieldLabels.value]: entity.utyd_unit_type,
-      [propertyMetadata.fuelTypeCode.fieldLabels.value]: entity.fyd_fuel_code,
-      [propertyMetadata.controlCode.fieldLabels.value]: entity.cyd_control_code,
-      [propertyMetadata.sourceCategoryDescription.fieldLabels.value]:
-        entity.uf_source_cat,
+      year: entity.pyd_op_year,

```

Comment:
Revert these changes since the applicable facility attributes endpoint isn't returning the correct data types anymore